src/genius/person.components/person-form/person-form.component.html

```diff
@@ -0,0 +1,100 @@
+      <input class="person-age person-input-edit" #age="ngModel" type="number" placeholder="Age" tabindex="4" [(ngModel)]="person.age" name="age" min="1" autocomplete="off">
+      <select class="person-sexlist" #sex="ngModel" name="sex" [(ngModel)]="person.sex" tabindex="5">
+        <ng-container *ngFor="let sex of PersonSex">
+          <option [value]="sex">{{sex}}</option>
+        </ng-container>
+      </select>
+      <div class="actions-wrap">
+        <button class="person-create-btn" type="button" (click)="savePerson()" routerLink="/Persons">Save</button>
+      </div>
+    </div>
+  </form>
+
+  <form *ngSwitchCase="personFormTemplateVersion.PERSON_SELECT" class="form-wrap">
+    <div class="wrap-select">
+      <div class="wrap-btn-close">
+        <button class="btn-close" (click)="closePersonSelect()" [routerLink] = "['/editFamily', familyId]" *ngIf="currentPersonFormPath === personFormPath.ADD_IN_EXIST_FAMILY">X</button>
+        <button class="btn-close" (click)="closePersonSelect()" routerLink="/createFamily" *ngIf="currentPersonFormPath === personFormPath.ADD_IN_NEW_FAMILY">X</button>
+      </div>
+      <div class="description-wrap">
+        <div class="person-title">{{personType.toUpperCase()}}</div>
+        <div class="person-description">
+          <input class="person-fn person-input-edit" #firstName="ngModel" type="text" [(ngModel)]="person.firstName" name="firstName" placeholder="First name" disabled *ngIf="person.firstName">
+          <input class="person-ln person-input-edit" type="text" placeholder="Last name" tabindex="2" [(ngModel)]="person.lastName" name="lastName" disabled *ngIf="person.lastName">
+          <input class="person-mn person-input-edit" type="text" placeholder="Middle name" tabindex="3" [(ngModel)]="person.middleName" name="middleName" disabled *ngIf="person.middleName">
+          <input class="person-age person-input-edit" #age="ngModel" type="number" placeholder="Age" [(ngModel)]="person.age" name="age" disabled *ngIf="person.age">
+        </div>
+      </div>
+      <div class="persons-list">
+        <div  [ngClass]="{'person-item-men': person.sex === personSex.MALE,'person-item-women': person.sex === personSex.FEMALE }" *ngFor = "let person of persons">
+          <person [person]="person" [templateVersion]="personTemplateType.FULL" (returnedPerson)="setChangePerson($event)"></person>
+        </div>
+      </div>
+      <div class="actions-wrap">
+        <button class="person-create-btn" type="button" (click)="selectPerson()"  [routerLink]="['/editFamily', familyId]" *ngIf="currentPersonFormPath === personFormPath.ADD_IN_EXIST_FAMILY">Select</button>
+        <button class="person-create-btn" type="button" (click)="selectPerson()"  routerLink="/createFamily" *ngIf="currentPersonFormPath === personFormPath.ADD_IN_NEW_FAMILY">Select</button>
```

Comment:
don't use ngIf for rendring logic

use angula binding for router
`<button class="person-create-btn" type="button" (click)="selectPerson()" [routerLink]="getRouterLink()">Select</button>`